example_test.go

```diff
@@ -1,54 +0,0 @@
-package quamina_test
```

Comment:
Why is was this file deleted?

Reply:
There were no tests in it and nothing called it, dead code. Probably a leftover forgotten by one of us.

Reply:
Ah, ok. Let me clarify: any file `example_*_test.go` in Go with functions prefixed with `Example*` inside serves as documentation examples rendered by Go doc, etc. See the current version for Quamina here: https://pkg.go.dev/github.com/timbray/quamina#example-New

Besides serving as documentation, these are also tests including assertions (see `// Output: pattern matched for event: "premium user"` inside the file). If you run our tests, those will also run and fail if the example is not consistent with the code anymore - that's a great benefit of the Go test and documentation tooling :).

Please revert the deletion.

Reply:
Oh, OK, I learned something today.